src/go/rpk/pkg/api/admin/admin.go

```diff
@@ -205,6 +205,13 @@ func send(
 
 	res, err := client.Do(req)
 
+	if err != nil {
+		if strings.Contains(err.Error(), "EOF") {
```

Comment:
could you log how the whole error looks like? It would not be obvious to me why this is enough to diagnose TLS error 🤔

Reply:
Yeah, it's not obvious at all. It seems the server terminates the connection over HTTP unexpectedly, because it expects HTTPS. I'll add a comment too.